docs/changelog/1862.md

```diff
@@ -0,0 +1 @@
+- Added configuration option for mixed convergence measurements: `<absolute-or-relative-convergence-measure abs-limit="1e-1" rel-limit="1.0" data="Data2" mesh="MeshOne"/>`.
```

Comment:
```suggestion
- Added a new mixed convergence measure `absolute-or-relative-convergence-measure`.
```
You did not only add the configuration option, but the actual functionality as well. And the exact config options are better looked up in the configuration reference.